frontend/src/components/TableComponents/TableCell.tsx

```diff
@@ -16,21 +16,37 @@ type TableCellProps = {
 const TableCell = (props: TableCellProps) => {
   const { data, index, first, last, tag, buttonHandler, ButtonModal } = props;
   if (first) {
+    const lastSpaceIndex = data?.lastIndexOf(' ');
+    const name = data?.substring(0, lastSpaceIndex); //student's name
+    const splitString = data?.split(' ');
+    const netId = splitString ? ' ' + splitString[splitString?.length-1] : '';
+
     return (<td
       key={index}
       className={cn(styles.passInfo, styles.cell, styles.firstCell)}>
-      {data}
+      <span style={{ fontWeight: 'bold' }}>{name}</span>
+      {netId}
```

Comment:
the first cell of a table isn't necessarily always going to be a name + netid, i think if you change the type of `data` to be ReactNode you can pass in JSX or strings